src/app/core/components/footer/footer.component.spec.ts

```diff
@@ -1,19 +1,20 @@
-import {async, ComponentFixture, TestBed} from '@angular/core/testing';
-
-import {FooterComponent} from './footer.component';
-import {RouterTestingModule} from '@angular/router/testing';
+import { ComponentFixture, TestBed } from '@angular/core/testing';
+import { FooterComponent } from './footer.component';
+import { RouterTestingModule } from '@angular/router/testing';
+import { AuthService } from './../../../shared/auth/auth-service/auth.service';
+import { MockAuthService } from './../../../../test/mocks/mock-auth.service';
 
 describe('FooterComponent', () => {
   let component: FooterComponent;
   let fixture: ComponentFixture<FooterComponent>;
 
-  beforeEach(async(() => {
-    TestBed.configureTestingModule({
+  beforeEach(async () => {
+    await TestBed.configureTestingModule({      
```

Comment:
Additional whitespace at the end of this line leads to linting issues.
Should have been caught by husky however 🤔 

<html>
<body>
<!--StartFragment-->

[Container] 2021/11/28 11:48:18 Running command yarn run lint
--
63 | yarn run v1.22.17
64 | $ ng lint
65 |  
66 | Linting "detektiv-kollektiv"...
67 |  
68 | /codebuild/output/src985833881/src/github.com/DetektivKollektiv/angular-ui/src/app/core/components/footer/footer.component.spec.ts
69 | 12:43  error  Trailing spaces not allowed  no-trailing-spaces
70 |  
71 | ✖ 1 problem (1 error, 0 warnings)
72 | 1 error and 0 warnings potentially fixable with the `--fix` option.
73 |  
74 | Lint errors found in the listed files.

<!--EndFragment-->
</body>
</html>

Reply:
Installed ESLint in VSCode to avoid such kind of errors.
Also eliminitad the whitespace, of course.

Reply:
Btw there are 11 spec files with deprecated use of `async `- that was the reason why I've updated the footer spec file.
Would propose a general update.

![grafik](https://user-images.githubusercontent.com/67347402/143949020-63bf2dea-b5fa-483c-be32-5cfdbb4a3078.png)